core/src/Modal.svelte

```diff
@@ -281,6 +283,11 @@
         )
       : '';
     window.focus();
+    setTimeout(() => {
```

Comment:
could you comment to this. Just an inline comment would be enough but also very helpful to understand